src/gui/torrentcreatordialog.cpp

```diff
@@ -170,7 +183,11 @@ void TorrentCreatorDialog::dragEnterEvent(QDragEnterEvent *event)
 // Main function that create a .torrent file
 void TorrentCreatorDialog::onCreateButtonClicked()
 {
-    const auto inputPath = Utils::Fs::toCanonicalPath(Path(m_ui->textInputPath->text().trimmed()));
+    auto inputPath = Path(m_ui->textInputPath->text().trimmed());
+#ifdef Q_OS_WIN
+    // Resolve the path in case it contains a shortcut (otherwise, the following usages will consider it invalid)
+    inputPath = Utils::Fs::toCanonicalPath(inputPath);
```

Comment:
```suggestion
#ifdef Q_OS_WIN
    // Resolve the path in case it contains a shortcut (otherwise, the following usages will consider it invalid)
    const auto inputPath = Utils::Fs::toCanonicalPath(Path(m_ui->textInputPath->text().trimmed()));
#else
    const auto inputPath = Path(m_ui->textInputPath->text().trimmed()); 
```